library/sync/src/main/java/org/cru/godtools/sync/work/SyncToolSharesWorker.kt

```diff
@@ -13,10 +13,10 @@ import org.cru.godtools.sync.task.ToolSyncTasks
 private const val WORK_NAME = "SyncToolShares"
 
 internal fun WorkManager.scheduleSyncToolSharesWork() =
-    enqueueUniqueWork(WORK_NAME, ExistingWorkPolicy.REPLACE, SyncWorkRequestBuilder<SyncToolSharesWorker>().build())
+    enqueueUniqueWork(WORK_NAME, ExistingWorkPolicy.KEEP, SyncWorkRequestBuilder<SyncToolSharesWorker>().build())
```

Comment:
What was the change to ```ExistingWorkPolicy``` needed for? Does it have something to do with the top-level ```getWorkManagerConfiguration``` we now have?

Reply:
nope, just a policy adjustment that made sense while I was adding the other network related tasks.

`REPLACE` will discard the existing work, `KEEP` will keep the existing work. The actual work being run would be the same between the old and new, the difference is in current backoff timers. I wanted to preserve the existing backoff timer if the code decides to re-schedule an existing work